src/Visualization/Python/GenerateVideo.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import click
+import glob
+import logging
+from paraview.simple import *
+
+#### disable automatic camera reset on 'Show'
+paraview.simple._DisableFirstRenderCameraReset()
+
+
+def generate_video(file_destination, volume_xmf, aha_xmf, ahb_xmf,
+                   output_destination, camera_angle):
+    """Generate Pictures from XMF files for BBH Visualizations
+
+    Generate pictures from BBH runs using the XMF files generated using
+    generate-xdmf. For files to be read properly, the XMF files being pointed to
+    should be in the same directory as the Volume.h5 and Surfaces.h5 files.
+
+    To splice all the pictures into a video, look into FFmpeg"""
+
+    # create a new 'XDMF Reader'
+    volulmeFilesxmf = XDMFReader(
```

Comment:
Please use snake_case for all variable names and function names, so this should be `volume_files_xmf`.